applications/asset_tracker_v2/sample.yaml

```diff
@@ -124,7 +124,6 @@ tests:
     platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
     integration_platforms:
       - nrf9160dk_nrf9160_ns
-      - thingy91_nrf9160_ns
```

Comment:
Is this removal related to nrf_modem 2.5.0 specifically, or just some clean up work?

Reply:
We are running out of space...
Agreed with @nordic-hani to remove thingy with the carrier overlay for now and re-enable it later if we can reduce the application size.